client/layout/logged-out.jsx

```diff
@@ -11,11 +11,8 @@ import { connect } from 'react-redux';
  * Internal dependencies
  */
 import MasterbarMinimal from 'layout/masterbar/minimal';
-import ThemesHead from 'my-sites/themes/head';
-import ThemeSheetComponent from 'my-sites/themes/sheet';
 
-const LayoutLoggedOutDesign = ( { routeName, match, section, hasSidebar, isFullScreen, tier = 'all' } ) => {
-	const primary = routeName === 'themes' ? <ThemeSheetComponent themeSlug={ match.theme_slug } /> : null;
+const LayoutLoggedOut = ( { primary, secondary, tertiary, section, hasSidebar, isFullScreen } ) => {
```

Comment:
Since `hasSidebar` and `isFullScreen` are not required props, should we have defaults for them? Seems like `hasSidebar = true` and `isFullScreen = false` would be the most common scenario.